Library/Homebrew/utils/inreplace.rb

```diff
@@ -50,7 +50,7 @@ def initialize(errors)
         block:        T.nilable(T.proc.params(s: StringInreplaceExtension).void),
       ).void
     }
-    def self.inreplace(paths, before = nil, after = nil, audit_result: true, &block)
+    def self.inreplace(paths, before = nil, after = nil, audit_result: false, &block)
```

Comment:
I don't think we should change this. When @cho-m talked about changing `audit_result` to `false`, I think they were referring to doing so in the affected formulae. We definitely don't want to change this globally.

Reply:
This makes more sense indeed. Updating